solc/CommandLineInterface.cpp

```diff
@@ -987,10 +987,13 @@ void CommandLineInterface::handleAst(string const& _argStr)
 		map<ASTNode const*, eth::GasMeter::GasConsumption> gasCosts;
 		// FIXME: shouldn't this be done for every contract?
 		if (m_compiler->runtimeAssemblyItems(m_compiler->lastContractName()))
-			gasCosts = GasEstimator::breakToStatementLevel(
+ 		{
+			auto ret = GasEstimator::breakToStatementLevel(
```

Comment:
If we go we merging this fix/workaround, please leave an explanatory comment here, otherwise this may be "optimized out" by future changes/people.